tests/helpers/testhelper.js

```diff
@@ -129,31 +129,6 @@ exports.setup.testRun = function (test, name) {
           return def.message === er.message;
         });
       });
-
-      test.ok(
-        undefinedErrors.length === 0 &&
-          unthrownErrors.length === 0 &&
-          wrongLineNumbers.length === 0 &&
-          duplicateErrors.length === 0,
```

Comment:
This change effectively disables the unit test suite. Since we want to continue
running tests for all the rest of the functionality, we'll have to be more
specific about *which* tests are invalidated by this change. Could you
re-introduce this code and instead remove the specific assertions that relate
to warning W041?